x-pack/legacy/plugins/ml/public/data_frame_analytics/pages/analytics_exploration/components/regression_exploration/_regression_exploration.scss

```diff
@@ -0,0 +1,3 @@
+.mlDataFrameAnalyticsRegression__evaluateStat span.euiToolTipAnchor {
```

Comment:
@alvarezmelissa87 Don't forget that you can pass custom classnames to EuiToolTip anchors with the prop `anchorClassName`. It's safer than targeting the `.eui` class directly.

Reply:
Thanks for the reminder 😄 Updated to use `anchorClassName` instead - 629d3fd022cdb4ae1048c39975f0c468a65a9f58